test/unit/ptable_test.rb

```diff
@@ -4,16 +4,19 @@ module ForemanTemplates
   class PtableTest < ActiveSupport::TestCase
     setup do
       # create a basic template with no OS assigned
-      @os_rh  = FactoryGirl.create(:operatingsystem, :family => 'Redhat')
-      @os_deb = FactoryGirl.create(:operatingsystem, :family => 'Debian')
-      @pt     = FactoryGirl.create(:ptable, :os_family => 'Redhat')
+      @os_rh   = FactoryGirl.create(:operatingsystem, :family => 'Redhat')
+      @os_deb  = FactoryGirl.create(:operatingsystem, :family => 'Debian')
+      @pt      = FactoryGirl.create(:ptable, :os_family => 'Redhat')
+      @new_org = FactoryGirl.create(:organization, :name => 'NewOrg')
+      @old_org = FactoryGirl.create(:organization, :name => 'OldOrg')
```

Comment:
You need this in provisioning_template_test too - in fact it's more important there :P